Spanner/src/Connection/Grpc.php

```diff
@@ -224,6 +234,8 @@ public function __construct(array $config = [])
         ];
         if ((bool) $config['emulatorHost']) {
             $grpcConfig += $this->emulatorGapicConfig($config['emulatorHost']);
+            $this->credentialsWrapper = new InsecureCredentialsWrapper();
```

Comment:
could I move these 2 lines to emulatorGapicConfig() as well? 
@bshaffer

Reply:
yes. you could, and then you could put the setting of `credentialsWrapper` outside the if:

```php
        if ((bool) $config['emulatorHost']) {
            $grpcConfig += $this->emulatorGapicConfig($config['emulatorHost']);
        } elseif (isset($config['apiEndpoint'])) {
            $grpcConfig['apiEndpoint'] = $config['apiEndpoint'];
        }
        $this->credentialsWrapper = $grpcConfig['credentials'];
```

Then you'd just have to set `$grpcConfig['credentials']` to `new InsecureCredentialsWrapper();` in `emulatorGapicConfig`.

Reply:
done